README.md

```diff
@@ -8,6 +8,27 @@ results. This allows builds to execute faster, both by reusing results already
 built by other clients and by allowing many actions to be executed in parallel,
 in excess of the resource limits of the machine running the build.
 
+There are a number of clients and services using these APIs, they are listed
+below.
+
+### Clients
+These tools use the Remote Execution API to distribute builds to workers.
+
+* [bazel](https://bazel.build)
```

Comment:
Why is only BuildStream capitalized? :-)

Reply:
I am acutely aware that the BuildStream maintainers are very sensitive to their capitalisation :) I always forget to capitalise Bazel because I think I am at the command line...

Will fix.